src/tracks/ui/BrushHandle.cpp

```diff
@@ -322,36 +338,31 @@ UIHandle::Result BrushHandle::Drag
          int yChange = 0;
          int radiusError = 0;
 
-//         if(mIsSmartSelection){
-//            // Correct the y coord (snap to highest energy freq. bin)
-//            long long startSC = posToLongLong(x0);
-//            wxInt64 targetFreq = PositionToFrequency(wt, 0, y0, mRect.y, mRect.height);
-//            if(auto *sView = dynamic_cast<SpectrumView*>(pView.get())){
-//               wxInt64 resFreq = SpectralDataManager::FindFrequencySnappingBin(wt,
-//                                                                               startSC,
-//                                                                               mFreqSnappingRatio,
-//                                                                               targetFreq);
-//               if(resFreq != - 1)
-//                  y0 = FrequencyToPosition(wt, resFreq, mRect.y, mRect.height);
-//            }
-//         }
+         int ym = y0;
+         if(mIsSmartSelection){
+            // Correct the y coord (snap to highest energy freq. bin)
+            long long startSC = posToLongLong(x0);
+            wxInt64 targetFreq = PositionToFrequency(wt, 0, ym, mRect.y, mRect.height);
+            if(auto *sView = dynamic_cast<SpectrumView*>(pView.get())){
+               wxInt64 resFreq = SpectralDataManager::FindFrequencySnappingBin(wt,
```

Comment:
I think you need to change this function now so it expects hop and bin values but it is still now expecting sample count and frequency values.